Allow obsolete isl-0.12.2 with custom gcc version

Copyright (C) 2017 Curt Brune <curt@cumulusnetworks.com>

SPDX-License-Identifier:     GPL-2.0

The older powerpc machines that use Linux kernel 3.2.69 require an
obsolete gcc-4.7.2, which is not a crosstool-ng default option.

gcc-4.7.2 requires a companion library, isl-0.12.2.

When using a custom gcc version in crosstool-ng, the system sets
CT_CC_GCC_6_or_later=y irrespective of the actual gcc version.

The isl-0.12.2 configuration option is unavailable when
CT_CC_GCC_6_or_later=y is true.

This patch relaxes the GCC version constraint on the isl-0.12.2
option.

diff --git a/config/companion_libs/isl.in b/config/companion_libs/isl.in
index bb2766f..1d59db5 100644
--- a/config/companion_libs/isl.in
+++ b/config/companion_libs/isl.in
@@ -52,7 +52,6 @@ config ISL_V_0_14
 config ISL_V_0_12_2
     bool
     prompt "0.12.2 (OBSOLETE)"
-    depends on !CC_GCC_6_or_later
     select ISL_V_0_12_or_later
     depends on OBSOLETE
 
